### PR TITLE
Update Simple Network Service package.

### LINKED ISSUE #203: Update the package Simple Network Service to add new features.
## This task is a part of the issue:
 - #122.

## Description
Update the `SimpleNetworkService` package in the `Woof` project after the class `NetworkService` was added into the package.

## DoD
The `SimpleNetworkService` package in the `Woof` project is up to date.

### PR DESCRIPTION
**Pull request by kirpichonok**
## Summary
This PR is a part of the work to resolve #203.

## Changes
- Update the `SimpleNetworkService` package in the `Woof` project up to date.

## Checks to complete
- [x] The PR title has a conventional commit style (short and descriptive).
- [x] The PR description includes essential information that helps understand the purpose of the changes.
- [x] I've built and run my changes, and no warnings or errors occurred.
- [x] All existing tests passed with my changes.
- [x] My code follows our style guide.
- [x] I've added tests for my code (if it makes sense).
- [x] PR has assignee, reviewers, milestone, it is properly linked to the project and to the issue.

### CODE REVIEW

File: Woof/WoofTests/ThirdParty/NetworkService/NetworkServiceIntegrationTests.swift [hunk 1]
```diff
@@ -3,6 +3,6 @@ import XCTest
 
 final class NetworkServiceIntegrationTests: XCTestCase {
     func testMethodTestUseFromNetworkServiceCanBeCalled() {
-        NetworkService().testUse()
+        _ = NetworkService<WoofAppEndpoint>.init()
```
Comment (NadzeyaShpakouskaya) [inline Woof/WoofTests/ThirdParty/NetworkService/NetworkServiceIntegrationTests.swift:6]: Why do we need to call init explicitly?
```suggestion
        _ = NetworkService<WoofAppEndpoint>()
```

**Review by NadzeyaShpakouskaya — CHANGES_REQUESTED** [review]
Please, update changelog file and consider small improvement:

File: Woof/WoofTests/ThirdParty/NetworkService/NetworkServiceIntegrationTests.swift [hunk 2]
```diff
@@ -3,6 +3,6 @@ import XCTest
 
 final class NetworkServiceIntegrationTests: XCTestCase {
     func testMethodTestUseFromNetworkServiceCanBeCalled() {
```
Comment (NadzeyaShpakouskaya) [inline Woof/WoofTests/ThirdParty/NetworkService/NetworkServiceIntegrationTests.swift:5]: Update test name as well

Comment (kirpichonok) [reply Woof/WoofTests/ThirdParty/NetworkService/NetworkServiceIntegrationTests.swift:5]: Fixed.

File: Woof/WoofTests/ThirdParty/NetworkService/NetworkServiceIntegrationTests.swift (hunk 1 above)
Comment (kirpichonok) [reply Woof/WoofTests/ThirdParty/NetworkService/NetworkServiceIntegrationTests.swift]: Fixed.

**Review by NadzeyaShpakouskaya — APPROVED** [review]
LGTM

**Review by iBYarchitect — APPROVED** [review]
LGTM

**Pull request merged**